python/src/trezorlib/messages.py

```
@@ -4602,6 +4627,122 @@ def __init__(
+        3: protobuf.Field("tuple", "EthereumABITupleInfo", repeated=False, required=False, default=None),
+        4: protobuf.Field("atomic_array", "EthereumABIType", repeated=False, required=False, default=None),
+        5: protobuf.Field("dynamic_array", "EthereumABIType", repeated=False, required=False, default=None),
+        6: protobuf.Field("tuple_array", "EthereumABITupleInfo", repeated=False, required=False, default=None),
+    }
+
+    def __init__(
+        self,
+        *,
+        atomic: Optional["EthereumABIType"] = None,
+        dynamic: Optional["EthereumABIType"] = None,
+        tuple: Optional["EthereumABITupleInfo"] = None,
```

Ruff (0.15.10)

[error] Function argument `tuple` is shadowing a Python builtin

(A002)